CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+# v0.19.3
+
+In **@liveblocks/client**:
+
+## Room.getStorageStatus
+
+Get the storage status.
+- `not-loaded`: Initial state when entering the room.
+- `loading`: Once the storage has been requested via room.getStorage().
+- `synchronizing`: When some local updates have not been acknowledged by Liveblocks servers.
+- `synchronized`: Storage is in sync with Liveblocks servers.
+
+## Room.subscribe("storage-status", status => { })
+
+Subscribe to storage status changes.
+
+Returns an unsubscribe function.
+
+```typescript
+room.subscribe("storage-status", (status) => {
+  switch(status) {
+     case "not-loaded":
```

Comment:
just a question about best practices- should this be "not-loaded" or empty?

Reply:
We went with `not-loaded` here to make it clear that the initial load for Storage hasn't even begun yet. Client that don't use Storage, for example, will always remain in this state. Once you successfully loaded the initial Storage state, it could be empty. That's why we wanted to avoid the term "empty" here. A loaded, but empty, state is still perfectly valid and would be "synchronized".